src/Mutate.test.tsx

```diff
@@ -358,6 +393,35 @@ describe("Mutate", () => {
 
       expect(onError.mock.calls.length).toEqual(0);
     });
+
+    it("should have the correct type definition", async () => {
```

Comment:
I don't understand this test case. There is no assertion. 🤔

Reply:
It's just to have a typescript error, jest fail if something is red ^^

Reply:
But I agree, I would like to have a better pattern to try this